src/Void/Init.fs

```diff
@@ -5,6 +5,19 @@ open Void.Lang.Interpreter
 open Void.Lang.Editor
 open Void.ViewModel
 
+type VoidOptions = {
+    enableVerboseMessageLogging : bool
+}
+
+module Options =
+    let verboseMessagesFlag = "--verbose-message-logging"
+
+    let parse (args : string[]) =
+        let verboseMessages = Array.tryFind (fun x -> x.Equals(verboseMessagesFlag)) args

```

Comment:
I like it. Keep the minimalism for the first pass. We may need to bring in an argument parsing library once this gets heavy.